src/Kernel/client/ExecutionPathVisualizer/circuit.ts

```diff
@@ -19,6 +19,15 @@ export interface Qubit {
     numChildren?: number;
 }
 
+export enum ConditionalRender {
+    /** Always rendered. */
+    None,
```

Comment:
I understand that this means "none" in the sense of "no condition", but maybe `Always` would be a clearer name for this, since the other values `OnZero` and `OnOne` also describe when to render the operation?

Reply:
Yea, that's a good point. Thanks!